.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea
 ebs-terraform/.terraform/
 efs-terraform/.terraform/
+###script temp files
+log
```

Comment:
But why don't you first delete the previously created resources, and then try again with the same name?

Reply:
I did, I ran the script twice, you're seeing both attempts,
note the message:

> We found that you already have a cluster provisioned with this script
> Do you want to remove it? (y/n)?y

in the second run. 

When I run again I always get the same error, the script doesn't delete my cluster.

Reply:
I'll try to drop any resource in my AWS setup and re-run again,
Could you share what might have caused these errors?

Reply:
Based on what I've seen here, it is possible that running the script from a different folder might have caused this.
This was answered to @galargh in a different comment - the script needs to be fixed (actually the paths to various files need to be fixed) so you can run it from any folder.
But for now, I believe that you need to `cd /eks` and then run the script from there until fixes are introduced.

So when you tried to create resources, the script failed because it couldn't reach the necessary files.
And then you tried to remove created resources, but the script couldn't retrieve resource IDs because it failed to create them in the first place.

Reply:
(this was fixed during a sync call, the script assumed some configuration in the `ENV` like default regions, the script will be updated to use what was actually configured in the `.env`).

Reply:
(resolved during the sync call)